openmetrics/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenMetrics Integration
+
+## Overview
+
+Extract custom metrics from any OpenMetrics endpoints.
+
+**Note:** All the metrics retrieved by this integration are considered as custom metrics.
```

Comment:
We could add a link to the custom metric doc: https://docs.datadoghq.com/developers/metrics/custom_metrics/